### ISSUE TITLE
Extend the RT's wb model to cover ProphotoRGB and account for dual illuminant DCP info

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#2242]
Originally reported on Google Code with ID 2258

```
http://code.google.com/p/rawtherapee/issues/detail?id=1987#c123
```

Reported by `michaelezra000` on 2014-02-21 20:16:31

**Comment by Beep6581** [comment]
@atorger has this been handled by one of your other issues?

**Comment by Beep6581** [comment]
Merged into #2099
